Formula/q.rb

```diff
@@ -1,30 +1,30 @@
-class Q < Formula
-  include Language::Python::Virtualenv
+# frozen_string_literal: true
 
```

Comment:
```suggestion
```
We don't include this in formulae.

Reply:
brew test failed on that one a year ago or something. I'll remove it of course.